src/m365/spo/commands/page/page-section-add.ts

```diff
@@ -154,11 +253,22 @@ class SpoPageSectionAddCommand extends SpoCommand {
         .sort();
       // zoneIndex for the new section to add
       const zoneIndex: number = this.getSectionIndex(zoneIndices, args.options.order);
+      let zoneId = undefined;
+      let backgroundControlToAdd: BackgroundControl | undefined = undefined;
```

Comment:
Try to leave some empty spaces between some logical sections to make it more readable.


```suggestion
      const zoneIndex: number = this.getSectionIndex(zoneIndices, args.options.order);
      
      let zoneId: string | undefined;
      let backgroundControlToAdd: BackgroundControl | undefined;
```

Reply:
Done